dojo_plugin/api/v1/ssh_key.py

```diff
@@ -53,3 +52,45 @@ def patch(self):
             )
 
         return {"success": True}
+    
+    
+@ssh_key_namespace.route("/delete")
+class DeleteKey(Resource):
```

Comment:
Let's not create another Resource, put this as a method named `delete` under the original resource. This gives us our CRUD operations for POST/DELETE to add/remove.